src/Item/Item.model.ts

```diff
@@ -33,13 +33,14 @@ export class Item extends Model<ItemAttributes> {
   }
 
   static findByThirdPartyIds(thirdPartyIds: string[]) {
+    const itemsTable = raw(this.tableName)
+    const collectionsTable = raw(Collection.tableName)
+
     return this.query<ItemAttributes>(SQL`
-      SELECT items.*
-        FROM ${raw(this.tableName)} items 
-        JOIN ${raw(
-          Collection.tableName
-        )} collections ON collections.id = item.collection_id
-        WHERE collections.third_party_id = ANY(${thirdPartyIds})`)
+      SELECT ${itemsTable}.*
+        FROM ${itemsTable}
+        JOIN ${collectionsTable} ON collections.id = ${itemsTable}.collection_id
```

Comment:
missing a ${collectionsTable} here right? (`collections.id`)

Question: it's a good idea to do this in general to avoid typos, but at the same time it makes the query really difficult to read. Can we strike a balance perhaps? shorter names but using the string in the query?

(we might need tests for queries heh)

Reply:
Ups, added it!
What makes me doubt is that we have the table name used as a variable for what I think is to avoid typos (or be reactive when the table changes), but we're only using it for the table once.